client/src/App.js

```diff
@@ -14,8 +18,22 @@ import {
   NotfoundPage,
   HomePage,
 } from "./Pages";
+import { setAuth, setLogout } from "./Store/Slices/checkAuthSlice";
 
 function App() {
+  const [cookies] = useCookies();
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    const checkToken = () => {
+      const { token } = cookies;
+      if (!token) dispatch(setLogout());
```

Comment:
Why you do this?!
As ther'e is no token So the state will keep the initial state,
the only place to `dispatch(setLogout())` at logout button